src/lib/useDataCollect.js

```diff
@@ -6,16 +9,17 @@ import {
   collectSearchWorkClick,
   collectSearchFeedback,
 } from "@/lib/api/datacollect.mutations";
-import { useFetcher } from "@/lib/api/api";
-
-let enabled = false;
 
-export function enableDataCollect(enable) {
-  enabled = enable;
+// For storybook/cypress test purposes
+let _dangerouslyForceConsent;
+export function dangerouslyForceConsent(consent) {
+  _dangerouslyForceConsent = consent;
```

Comment:
For en sikkerheds så synes jeg du bør tjekke på `process.env.STORYBOOK_ACTIVE`, så vi er helt sikre på at vi ikke kommer til at bruge den der funktion forkert... Eller hvad?